code/game/objects/structures/crates_lockers/closets/secure/hydroponics.dm

```diff
@@ -11,3 +11,90 @@
 	new /obj/item/cultivator(src)
 	new /obj/item/hatchet(src)
 	new /obj/item/secateurs(src)
+
+//Plocker
+/obj/structure/closet/secure_closet/plocker
```

Comment:
plocker is a confusing path if anyone wanted to spawn it, why not just secure_closet/peach

Reply:
True, renamed plocker obj path to peach.